packages/data/src/redux-store/metadata/actions.js

```diff
@@ -68,6 +87,26 @@ export function finishResolutions( selectorName, args ) {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that a batch of selector resolutions has
+ * completed and at least one of them has failed.
+ *
+ * @param {string}         selectorName Name of selector for which resolver triggered.
+ * @param {unknown[]}      args         Array of arguments to associate for uniqueness, each item
+ *                                      is associated to a resolution.
+ * @param {(Error|null)[]} errors       Array of errors to associate for uniqueness, each item
+ *                                      is associated to a resolution.
+ * @return {{ type: 'FAIL_RESOLUTIONS', selectorName: string, args: unknown[], errors: Array<Error|null> }} Action object.
```

Comment:
Can there be a failure that has a `null` error? I'd expect that where there is a failure, there is always an error object, or isn't there?

Reply:
Yes, see https://github.com/WordPress/gutenberg/pull/38669#discussion_r809148493

Perhaps this should even be `Array<any>` since we cannot guarantee the data type.

Reply:
I went for `Error | unknown`